### ISSUE TITLE
Display first Nominatim search result automatically

### ISSUE THREAD

**Issue by Dominik-K** [openstreetmap/openstreetmap-website#919]
If I search for something, like "Berlin Hauptbahnhof", then usually the first result from Nominatim is the right one. Can you display the first search result automatically?

**Comment by tomhughes** [comment]
Of course we can, the correct question is should we? Historically we have always thought it was better not to, in part because there are potentially multiple providers loading in parallel so the definition of "first" is non-deterministic.

Pretty sure there is already a ticket for this anyway, though it may be in trac as I can't immediately see one here.

**Comment by Dominik-K** [comment]
I did not find a Trac issue. My experience with search on OSM website is that only Nominatim shows useful results, so I would vote for displaying Nominatim search result.

**Comment by neiljp** [comment]
@tomhughes I can certainly see the parallel-loading issue, though is this not just nominatim & geonames? Agreed that the problem would occur with simply more than one, though. Perhaps a user settings preference could resolve this?

@Dominik-K Whether each set is more useful will depend on the location in the world, I suspect.

Personally, my suggestion would be to take the 'top' results (before 'more' is clicked) and do a bbox of them with each marked with a (small) marker, which is then highlighted on hover (or scroll of a 'selected' item?) on a given name. This issue could then be solved (perhaps) by having a 'default' of selecting/highlighting the top item (with a preference setting?). Generally re the multiple lists loading in parallel, this could reload/zoom the bbox depending upon load times, and simply highlight more when the next set is loaded. I can move this idea to a separate issue, if it is not considered completely unfeasible already.

**Comment by adius** [comment]
This would be so great!

**Comment by Dominik-K** [comment]
Thanks @tomhughes for your commit. 
However, I like @neiljp's idea as well. Shell we create a new issue for it?
